### PR TITLE
add option withDeleted to be able to populate with deleted objects

### PR DESCRIPTION
**Pull request by svengau**
this PR is a proposed solution for the following issue: https://github.com/dsanel/mongoose-delete/issues/21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +210,18 @@ Pet.findWithDeleted(function (err, documents) {
 
 ```
 
+### Populate deleted documents if overrideMethods is distribute
+
+We have the options `withDeleteded` that can be pass to the populate() function.
```
Comment (giorni) [inline README.md]: Typo.

**Comment by alejandro-martinez** [comment]
any update on this? I need this feature

**Comment by tanoso** [comment]
@giorni @svengau What happened with this one? I need it, I would really appreciate it if it could be merged.

File: README.md (hunk 1 above)
Comment (svengau) [reply README.md]: fixed.

**Comment by svengau** [comment]
@tanoso Sorry I left this PR lying around. I've just fixed the coverage code and the typo reported by @giorni.

**Comment by seeARMS** [comment]
This would be a huge help for me as well.

**Comment by minhuyen** [comment]
How is the pull request? @svengau Can you resolve the conflict and pull request again! Thanks a lot. I need this feature for my project. I would really appreciate it if it could be merged.

**Comment by svengau** [comment]
@minhuyen this PR was ready to be merged, before @dsanel added new commits in the master. 

@dsanel please, could you confirm you are OK to merge it, once the conflict resolved ?

**Comment by AbdulHashik0780** [comment]
> @minhuyen this PR was ready to be merged, before @dsanel added new commits in the master.
> 
> @dsanel please, could you confirm you are OK to merge it, once the conflict resolved ?

any updates ? This would be a great feature.

**Comment by Kuzzy** [comment]
Up!

**Comment by marcotuna** [comment]
@svengau can you resolve the index.js conflict?
It would be useful to have this merged, I've been looking for this.

**Comment by sayhicoelho** [comment]
Any updates?

**Comment by PrinceAnsong** [comment]
Desperately in need of this.

**Comment by dsanel** [comment]
[v0.5.3](https://github.com/dsanel/mongoose-delete/blob/master/CHANGELOG.md#v053)

**Pull request closed without merging**